modules/@apostrophecms/asset/index.js

```diff
@@ -360,43 +384,49 @@ module.exports = {
             const jsModules = `${scene}-module-bundle.js`;
             const jsNoModules = `${scene}-nomodule-bundle.js`;
             const css = `${scene}-bundle.css`;
-            fs.writeFileSync(
-              `${bundleDir}/${jsModules}`,
-              Object.entries(self.builds).filter(
-                ([ name, options ]) => options.scenes.includes(scene) &&
-                options.outputs.includes('js') &&
-                (!options.condition || options.condition === 'module')
-              ).map(([ name, options ]) => {
-                return `// BUILD: ${name}\n` + fs.readFileSync(`${bundleDir}/${name}-build.js`, 'utf8');
-              }).join('\n')
-            );
-            fs.writeFileSync(
-              `${bundleDir}/${jsNoModules}`,
-              Object.entries(self.builds).filter(
-                ([ name, options ]) => options.scenes.includes(scene) &&
-                options.outputs.includes('js') &&
-                (!options.condition || options.condition === 'nomodule')
-              ).map(([ name, options ]) => {
-                return `// BUILD: ${name}\n` + fs.readFileSync(`${bundleDir}/${name}-build.js`, 'utf8');
-              }).join('\n')
-            );
-            fs.writeFileSync(
-              `${bundleDir}/${css}`,
-              Object.entries(self.builds).filter(
-                ([ name, options ]) => options.scenes.includes(scene) &&
-                options.outputs.includes('css')
-              ).map(([ name, options ]) => {
-                const file = `${bundleDir}/${name}-build.css`;
-                if (fs.existsSync(file)) {
-                  return `/* BUILD: ${name} */\n` + fs.readFileSync(file, 'utf8');
-                } else {
-                  return '';
-                }
-              }).join('\n')
-            );
+            writeAssetFile({
```

Comment:
I thought originally that I had to write some files from this function for my extra bundles, that's why I refactored this part. I realized later that it wasn't needed.
Kept the new version since it avoid code redundancy and make it more readable.